docs/PytorchXLA_overview.md

```diff
@@ -0,0 +1,270 @@
+### **Objective:**
+This document provides a high-level overview of PyTorch XLA and illustrates a
+few examples how PyTorch code is converted to run on XLA devices (e.g. TPUs).
+This is not a complete solution, and additional changes may be required
+depending on the specific code. However, this document should serve as a
+starting point for the conversion process.
+
```

Comment:
can you add this doc to https://github.com/pytorch/xla/blob/master/docs/source/index.rst#test? Then it will start showing up in https://pytorch.org/xla/release/2.0/index.html You can put it above trouble shooting

Reply:
Done, thanks.